content/en/news/releases/1.21.x/announcing-1.21/_index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Announcing Istio 1.21.0
+linktitle: 1.21.0
+subtitle: Major Release
+description: Istio 1.21 Release Announcement.
+publishdate: 2024-02-23
+release: 1.21.0
+aliases:
+- /news/announcing-1.21
+- /news/announcing-1.21.0
+---
+
+We are pleased to announce the release of Istio 1.21. This is the first Istio release of 2024. We would like to thank the
+entire Istio community for helping get the 1.21.0 release published. We would like to thank the Release Managers for
+this release, `Aryan Gupta` from Google, `Jianpeng He` from Tetrate, and `Sumit Vij`. The release
+managers would specially like to thank the Test & Release WG lead Eric Van Norman (IBM) for his help and guidance
+throughout the release cycle. We would also like to thank the maintainers of the Istio work groups and the broader Istio
+community for helping us throughout the release process with timely feedback, reviews, community testing and for all
+your support to help ensure a timely release.
+
+{{< relnote >}}
+
+{{< tip >}}
+Istio 1.21.0 is officially supported on Kubernetes versions `1.26` to `1.29`.
+{{< /tip >}}
+
+## What's new
+
+### Easing upgrades with compatibility versions
+
+Istio 1.21 introduces a new concept known as [_compatibility versions_]
+(/docs/setup/additional-setup/compatibility-versions/).
+
+Compatibility versions solves a long running problem in Istio: as time passes,
+changes to the behavior of Istio may be desired to fix bugs, improve integration with the rest of the
+ecosystem, improve security, or fix surprising behaviors. However, even the smallest
+behavioral changes can cause issues on upgrade for a project like Istio deployed across
+thousands of companies in production. At best, this makes upgrades more challenging - at
+worst, it pushes users to not upgrade at all!
+
+With compatibility versions, behavioral changes are decoupled from the Istio version. For
+example, if you want to upgrade to Istio 1.21 but don't want to adopt the changes
+introduced yet, simply install with `--set compatibilityVersion=1.20` to retain the 1.20
+behavior.
+
+Not sure if you need the old behavior? Not a problem, `istioctl` can tell you!
+
+{{< text shell >}}
+$ istioctl x precheck --from-version {{< istio_previous_version >}}
```

Comment:
Not sure if we should use `x` or `experimental`. Also below.

Reply:
We typically want experimental in change notes, so I think for consistency we should use the full command name over alias.